### DISCUSSION TITLE
Organization#base/type - Типы организаций

### DISCUSSION THREAD

**Discussion by niquola** [fhir-ru/core#45] [Административная РГ]
Нужно ли кодировать тип организации?

Предложено в https://github.com/fhir-ru/core/discussions/26:

* ЛО - лицензируемая организация , те новые организации которые собирают данные для получения лицензии (ещё не МО, ФО и ОО)
* МО - Мед.Организация
* ФО - Фармацевтическая Организация
* ОО- образовательная организация , они уже сейчас есть в ФРМО
* ОЗ - Организатор Здравоохранения

Есть ли словарь для этого в https://nsi.rosminzdrav.ru/?

**Comment by Vrodvi** [comment]
Если помимо медицинских организаций учитываются образовательные и фармацевтические, то возможно стоит добавить ещё и страховые организации?
HL7 выделяет страховые организации в справочнике типов организаций - [valueset-organization-type](https://www.hl7.org/fhir/valueset-organization-type.html)

**Comment by eugenekogan-spb** [comment]
Коллеги, видимо, предложенная выше классификация организаций (лицензируемая, медицинская, страховая и т.п.) не основывается на опубликованном справочнике. В НСИ ЕГИСЗ я ее не нашел. Классификация взята из рабочей таблички ФРМО https://docs.google.com/spreadsheets/d/1GGGzZTmJ7yVrEUCTu9AO8Q1epMoaZ08C/edit#gid=2043079596 . Как пояснял @iradio, это не текущая структура ФРМО, а планируемая (в разработке).  Можем ли мы опираться на список, у которого нет источника?

**Reply by Vrodvi** [reply]
Вопрос здесь в авторстве и ответственности за используемые справочники.
Считаю что все используемые в профилях справочники должны браться из существующих источников, т.к. если сообщество FHIR-ru самостоятельно определит справочник, то может возникнуть ситуация с технической\организационной невозможностью его поддерживать, что сделает использование профилей также невозможным.

Поэтому справочник типов организаций обязательно должен быть заведён в ФНСИ.
В руководстве же мы можем заранее завести CodeSystem с необходимым набором кодов как временное решение, в котором будет заготовлена "заглушка" для указания ссылки на ФНСИ, когда справочник там будет заведён.
